cheatsheets/Clickjacking_Defense_Cheat_Sheet.md

```diff
@@ -87,11 +93,11 @@ This approach is discussed on the [JavaScript.info website](https://javascript.i
 
 ## Limitations
 
-As of 2019 the `SameSite` attribute is supported by [most modern browsers](https://caniuse.com/#feat=same-site-cookie-attribute), although it it not widely supported on mobile browsers. Support in Internet Explorer will varies depending on the updates installed and the underlying operating system.
+If the Clickjacking attack does not require the user to be authenticated, this attribute will not provide any protection.
 
-Additionally, if the Clickjacking attack does not require that the user is authenticated, this attribute will not provide any degree of protection.
+Additionally, while `SameSite` attribute is supported by [most modern browsers](https://caniuse.com/#feat=same-site-cookie-attribute), there are still a significant number of users (approximately 12% as of August 2019) with browsers which do not support it.
```

Comment:
there `is` still ... and not are

Reply:
"number" is a plural here, so I believe "are" is correct:

https://www.grammar-monster.com/lessons/singular_plural_number.htm

Reply:
The use of “is” and “are” depends on whether the thing being referred to is singular or plural; the question in this case is whether we mean to say “a high number of users” (in which case, we use “is” to refer to the numerical value - see the link @rbsec provided) or if we mean to say “many users” (in which case, the sentence should be reworded).

I suggest the following options for revision:

> ... there is still a significant number of users...

> ... there are still many users...

On an unrelated note, the latter part of this sentence should read, “with browsers that do not support it.” See [_that_ vs _which_](https://www.quickanddirtytips.com/education/grammar/which-versus-that-0).

This has been a PSA from your friendly neighbourhood grammarist 👋

Reply:
@victoriadrake thanks - I've updated it to `many users` and also amended the final sentence.